doc/team_practices.md

```diff
@@ -42,20 +42,40 @@ See the [story tracker documentation](https://www.pivotaltracker.com/help/api?ve
 
 1. Click **Finish** on the story in the story tracker.
 
-1. Someone from the team should review the PR. If you do not get feedback within 24 hours, assign the PR to a team member.
-The Reviewer is encouraged to indicate a "Ship it" (or your favorite Ship It emoticon) before merging the pull request. 
-Consider pointing out the awesomeness of your teammate's code, too.
-
-1. The Reviewer merges the PR. Pro tip: If the PR title follows the conventions above, 
+1. Someone from the team should review the PR. If you do not get feedback within 24 hours, 
+assign the PR to a team member for review. The Reviewer should consider the automated
+testing status and Code Climate reports, in addition to checking the code for architectural
+consistency, style and legibility. Code reviews are encouraged early in the development process;
+you can always create a PR with a `[WIP]` prefix before you are ready to deliver, and ask for review
+of work-in-progress.
+
+1. The Reviewer should also test the code. This can be done on the `c2-dev` or `c2-staging` environment.
+Check with your teammates to see which environment might already be in use.
+Example flow:
+
+  1. `% cd /tmp && mkdir deploy-qa && cd deploy-qa`
+  1. `% git clone git@github.com:18F/C2.git`
+  1. `% cd C2`
+  1. `% git checkout -b qa-123456-fix-timezones`
+  1. `% git merge -m 'temp qa branch' origin/123456-fix-timezones`
+  1. `% script/deploy c2-dev`
```

Comment:
Minor: I think this would look better as a [fenced code block](https://gist.github.com/clintel/1155906).